container_test.go

```diff
@@ -303,6 +304,71 @@ func Test_BuildImageWithContexts(t *testing.T) {
 	}
 }
 
+func TestCustomLabelsImage(t *testing.T) {
+	// --- Given ---
+	const (
+		myLabelName  = "org.my.label"
+		myLabelValue = "my-label-value"
+	)
+
+	ctx := context.Background()
+	req := testcontainers.GenericContainerRequest{
+		ContainerRequest: testcontainers.ContainerRequest{
+			Image:  "alpine:latest",
+			Labels: map[string]string{myLabelName: myLabelValue},
+		},
+	}
+
+	// --- When ---
+	ctr, err := testcontainers.GenericContainer(ctx, req)
+
+	// --- Then ---
+	require.NoError(t, err)
+	t.Cleanup(func() { assert.NoError(t, ctr.Terminate(ctx)) })
+
+	ctrJSON, err := ctr.Inspect(ctx)
+	require.NoError(t, err)
+	assert.Equal(t, myLabelValue, ctrJSON.Config.Labels[myLabelName])
+}
+
+func TestCustomLabelsBuildOptionsModifier(t *testing.T) {
+	// --- Given ---
+	const (
+		myLabelName        = "org.my.label"
+		myLabelValue       = "my-label-value"
+		myBuildOptionLabel = "org.my.bo.label"
+		myBuildOptionValue = "my-bo-label-value"
+	)
+
+	ctx := context.Background()
+	req := testcontainers.GenericContainerRequest{
+		ContainerRequest: testcontainers.ContainerRequest{
+			FromDockerfile: testcontainers.FromDockerfile{
+				Context:    "./testdata",
+				Dockerfile: "Dockerfile",
+				BuildOptionsModifier: func(opts *types.ImageBuildOptions) {
+					opts.Labels = map[string]string{
+						myBuildOptionLabel: myBuildOptionValue,
+					}
+				},
+			},
+			Labels: map[string]string{myLabelName: myLabelValue},
+		},
+	}
+
+	// --- When ---
+	ctr, err := testcontainers.GenericContainer(ctx, req)
+
+	// --- Then ---
+	require.NoError(t, err)
+	t.Cleanup(func() { assert.NoError(t, ctr.Terminate(ctx)) })
+
+	ctrJSON, err := ctr.Inspect(ctx)
+	require.NoError(t, err)
+	assert.Equal(t, myLabelValue, ctrJSON.Config.Labels[myLabelName])
+	assert.Equal(t, myBuildOptionValue, ctrJSON.Config.Labels[myBuildOptionLabel])
```

Comment:
Let's use require here too
```suggestion
	require.Equal(t, myLabelValue, ctrJSON.Config.Labels[myLabelName])
	require.Equal(t, myBuildOptionValue, ctrJSON.Config.Labels[myBuildOptionLabel])
```